spec/factories/listing_amenity.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :listing_amenity do
+    title { 'MyString' }
+    description { 'MyText' }
+    is_available { false }
+    metadata { 'MyText' }
+    listing { nil }
```

Comment:
create an actual listing relationship

Reply:
Will be handled after PR https://github.com/andykimchris/inicro-backend/pull/9 is saved